server/server.js

```diff
@@ -89,11 +89,21 @@ app.post(
   }
 );
 
+app.delete(
+  "/logout",
+  sessionController.terminateSession,
+  (req, res) => {
+    console.log("completing delete request to '/logout");
+    res.status(200).json(req.session);
+  }
+)
+
 app.use('/sessionTest',
+  // sessionController.startSession,
   sessionController.isLoggedIn,
   (req, res) => {
-    console.log('user isLoggedIn successfully.  Returning status 418');
-    res.sendStatus(418);
+    console.log("completing request to '/sessionTest");
+    res.status(200).json(req.session);
```

Comment:
same here

Reply:
Was used for testing purposes.  Suggested change implemented.